homeassistant/generated/zeroconf.py

```diff
@@ -4,65 +4,242 @@
-    "Abode": "abode",
-    "BSB002": "hue",
-    "C105X": "roku",
-    "C135X": "roku",
-    "EB-*": "ecobee",
-    "Escea": "escea",
-    "HHKBridge*": "hive",
-    "Healty Home Coach": "netatmo",
-    "Iota": "abode",
-    "LIFX A19": "lifx",
-    "LIFX BR30": "lifx",
-    "LIFX Beam": "lifx",
-    "LIFX Candle": "lifx",
-    "LIFX Clean": "lifx",
-    "LIFX Color": "lifx",
-    "LIFX DLCOL": "lifx",
-    "LIFX DLWW": "lifx",
-    "LIFX Dlight": "lifx",
-    "LIFX Downlight": "lifx",
-    "LIFX Filament": "lifx",
-    "LIFX GU10": "lifx",
-    "LIFX Lightstrip": "lifx",
-    "LIFX Mini": "lifx",
-    "LIFX Nightvision": "lifx",
-    "LIFX Pls": "lifx",
-    "LIFX Plus": "lifx",
-    "LIFX Tile": "lifx",
-    "LIFX White": "lifx",
-    "LIFX Z": "lifx",
-    "MYQ": "myq",
-    "NL29": "nanoleaf",
-    "NL42": "nanoleaf",
-    "NL47": "nanoleaf",
-    "NL48": "nanoleaf",
-    "NL52": "nanoleaf",
-    "NL59": "nanoleaf",
-    "Netatmo Relay": "netatmo",
-    "PowerView": "hunterdouglas_powerview",
-    "Presence": "netatmo",
-    "Rachio": "rachio",
-    "SPK5": "rainmachine",
-    "Sensibo": "sensibo",
-    "Smart Bridge": "lutron_caseta",
-    "Socket": "wemo",
-    "TRADFRI": "tradfri",
-    "Touch HD": "rainmachine",
-    "Welcome": "netatmo",
-    "Wemo": "wemo",
-    "YL*": "yeelight",
-    "ecobee*": "ecobee",
-    "iSmartGate": "gogogate2",
-    "iZone": "izone",
-    "tado": "tado",
+    "3810X": {
+        "domain": "roku",
+        "iot_class": "local_polling",
+    },
+    "3820X": {
+        "domain": "roku",
+        "iot_class": "local_polling",
+    },
+    "4660X": {
+        "domain": "roku",
+        "iot_class": "local_polling",
+    },
+    "7820X": {
+        "domain": "roku",
+        "iot_class": "local_polling",
+    },
+    "819LMB": {
+        "domain": "myq",
+        "iot_class": "cloud_polling",
+    },
+    "AC02": {
+        "domain": "tado",
+        "iot_class": "cloud_polling",
+    },
+    "Abode": {
+        "domain": "abode",
+        "iot_class": "cloud_push",
+    },
+    "BSB002": {
+        "domain": "hue",
+        "iot_class": "local_push",
+    },
+    "C105X": {
+        "domain": "roku",
+        "iot_class": "local_polling",
+    },
+    "C135X": {
+        "domain": "roku",
+        "iot_class": "local_polling",
+    },
+    "EB-*": {
+        "domain": "ecobee",
+        "iot_class": "cloud_polling",
+    },
+    "Escea": {
+        "domain": "escea",
+        "iot_class": "local_push",
+    },
+    "HHKBridge*": {
+        "domain": "hive",
+        "iot_class": "cloud_polling",
+    },
+    "Healty Home Coach": {
+        "domain": "netatmo",
+        "iot_class": "cloud_polling",
+    },
+    "Iota": {
+        "domain": "abode",
+        "iot_class": "cloud_push",
+    },
+    "LIFX A19": {
+        "domain": "lifx",
+        "iot_class": "local_polling",
+    },
+    "LIFX BR30": {
+        "domain": "lifx",
+        "iot_class": "local_polling",
+    },
+    "LIFX Beam": {
```

Comment:
Out of scope for this PR:

Since we support wildcard, could we rewrite this to `"LIFX *"` ?

Reply:
We can't since some models aren't supported by the lifx integration but are supported by homekit